<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1107</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="int_time_label">
   <property name="geometry">
    <rect>
     <x>154</x>
     <y>90</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Integration Time (ms):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="intTimeInput">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>90</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>231</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Capture Spectrum</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>231</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>STOP</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="plot_graph">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>210</y>
     <width>1111</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="extTriggerCheckBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Enable External Trigger</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SaveData_dir">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>170</y>
     <width>431</width>
     <height>22</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="browseButton">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>120</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLabel" name="int_time_label_2">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>150</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Selected Directory Path:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
